scripts/check_changes_in_right_version.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
we have a lot of bash scripts in the top-level folder that we should probably move here too

Reply:
(doesn't have to happen in this PR, just thinking out loud)

Reply:
yep, just thought the same when i saw the scripts so i started on a new folder